hack/update-deps.sh

```diff
@@ -13,6 +13,9 @@ cd "${ROOT_DIR}"
 
 # This controls the knative release version we track.
 KN_VERSION="release-0.20"
+EVENTING_VERSION="release-v0.20.0"
+EVENTING_KAFKA_VERSION="release-v0.20.0"
+SERVING_VERSION="release-v0.20.0"
```

Comment:
Those should be read from `project.yaml`, right?

Reply:
Maybe. As long as the version in project.yaml always matches the given branch in those repositories. This would enforce it. But this would also force the team to always do the upgrade at once. As I heard right now it might be done with at least two PRs - one for Golang dependencies, one for yaml changes etc (like here: https://github.com/openshift-knative/serverless-operator/pull/791/files). I don't want to change something that has been working well so far. Will defer to @markusthoemmes

Reply:
Yeah let's defer that. It's generally useful to break these changes apart imo.

Reply:
I just hate having different, conflicting, sources of truth :anger:

Reply:
Everybody does, but the Golang deps aer usually their own can of worms when it comes to upgrading versions and they usually don't stricrtly have to move in lockstep to the component versions themselves.